dwh/query_lib_test.py

```diff
@@ -14,95 +14,95 @@
 
 import unittest
 
-import query_lib
-
+import query_lib_spark as query_lib
```

Comment:
nit:
```suggestion
import query_lib_spark
```
and replace all usage.